src/server/apis/dash-data.ts

```diff
@@ -32,14 +32,16 @@ export class DashData {
 
   async fetchUserData(login: string, token: string): Promise<api.DashResponse> {
     const openPrQuery = 'is:open is:pr archived:false';
+    const reviewedQueryString =
+        `reviewed-by:${login} ${openPrQuery} -author:${login}`;
 
     const viewerPrsResult = await this.github.query<ViewerPullRequestsQuery>({
       query: prsQuery,
       variables: {
         login,
         reviewRequestsQueryString: `review-requested:${login} ${openPrQuery}`,
-        reviewedQueryString:
-            `reviewed-by:${login} ${openPrQuery} -author:${login}`,
+        reviewedQueryString,
+        mentionsQueryString: `${reviewedQueryString} mentions:${login}`,
```

Comment:
would be nice if this followed the same pattern and `reviewedQueryString` and gets defined outside of this object.

Reply:
Done